media/js/firefox/new/scene1_fxa_modal.js

```diff
@@ -8,7 +8,7 @@
     'use strict';
 
     var client = window.Mozilla.Client;
-    var $downloadButton = $('#download-button-desktop-release .download-link[data-download-os="Desktop"]');
+    var $downloadButton = $('.download-button .download-link[data-download-os="Desktop"]');
```

Comment:
Note for reviewer: this change makes both the primary and secondary download buttons on `/de/firefox/new/` and `/en-GB/firefox/new/` trigger the FxA modal for signed-out Firefox users.